FreeTAKServer/controllers/configuration/MainConfig.py

```diff
@@ -1,18 +1,17 @@
 import os
+import sys
 import re
 import yaml
 currentPath = os.path.dirname(os.path.abspath(__file__))
+
 from pathlib import Path
 from string import ascii_letters, digits, punctuation
 from uuid import uuid4
 
 # the version information of the server (recommended to leave as default)
-FTS_VERSION = 'FreeTAKServer-1.9.9.6 Public'
+FTS_VERSION = 'FreeTAKServer-1.9.9.7 Public'
 API_VERSION = '1.9.5'
-# TODO Need to find a better way to determine python version at runtime
-PYTHON_VERSION = 'python3.8'
-USERPATH = '/usr/local/lib/'
-MAINPATH = fr'{USERPATH}{PYTHON_VERSION}/dist-packages/FreeTAKServer'
+PERSISTENCE_PATH = r'/opt/fts/'
```

Comment:
Thought, what if we make this value solely an environment variable rather then a yaml config variable? Since the config overwrite does not work correctly, grabbing this from the environment beforehand will solve the issues we are having where accessing any attribute that relies on this value.

Otherwise we will need to use the template system @naman108 mentioned